src/components/DaySelector.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import styles from '../styles';
+// Define a type for the props expected by DaySelector
+type DaySelectorProps = {
```

Comment:
I'm not sure if its different in react native but at my work we would put the validation props under the component and not on top.

Reply:
maybe we'll consult with @manu about this

Reply:
I think it doesn't really matter, it's a convention that u should do what suits you.
From what I saw so far there were 2 options:
* On top of the file
* In a separate file - I actually prefer this, I think it's much cleaner.  Like a folder called `models` in parallel to `components` where you have files for different types and interfaces and stuff like that